hack/lib/common.sh

```diff
@@ -798,12 +798,18 @@ EOF
     fi >>/tmp/kube-proxy.yaml
 
     kube::common::generate_kubeproxy_certs
+    local port=${API_SECURE_PORT}
+    local protocol="https"
+    if [[ "${IS_SCALE_OUT}" == "true" ]]; then
```

Comment:
probably a comment with TODO can help future work to enable the secure connection on scale-out env in local-up.

Reply:
I think whoever reenable SSL mode in local cluster up will check all the kube config and override values. There is no plan to do so in the near future, so prefer not leave a TODO here and there.